lib/internal/modules/esm/hooks.js

```diff
@@ -553,15 +570,30 @@ class HooksProxy {
     }
   }
 
-  async makeAsyncRequest(method, ...args) {
+  /**
+   * Invoke a remote method asynchronously.
+   * @param {any[]} [transferList] Objects in `args` to be transferred
+   * @param {string} method Method to invoke
+   * @param  {any[]} args Arguments to pass to `method`
+   * @returns {Promise<any>}
+   */
+  async makeAsyncRequest(transferList, method, ...args) {
```

Comment:
This still needs refactoring so that `method` is first again. Likewise for `makeSyncRequest` and other places where `transferList` was added.

Reply:
Ah ok sorry I didn't realize that had been decided on. I can do that though.